spec/requests/account_suspended_spec.rb

```diff
@@ -0,0 +1,91 @@
+require "spec_helper"
+
+RSpec.describe "When account is enabled", type: :request do
+  before do
+    allow(Rails.configuration).to receive(:feature_flag_govuk_accounts).and_return(true)
+  end
+
+  context "and accounts is returning a 503 error code" do
+    before { stub_request(:get, Services.accounts_api).to_return(status: 503) }
+
+    describe "/login" do
+      it "redirects a user to the accounts 503 error page" do
+        get transition_checker_new_session_path
+
+        expect(response).to redirect_to(Services.accounts_api)
+      end
+    end
+
+    describe "/login/callback" do
+      it "redirects a user to the accounts 503 error page" do
+        get transition_checker_new_session_callback_path
+
+        expect(response).to redirect_to(Services.accounts_api)
+      end
+    end
+
+    describe "/logout" do
+      it "redirects a user to the accounts 503 error page" do
+        get transition_checker_end_session_path
+
+        expect(response.body).to eq("Redirecting to #{Services.accounts_api}/sign-out?continue=1")
```

Comment:
i suspect the parameters here are much a muchness, 
but i've made a special case of `/logout`, I figure in that scenario we _do_ want to delete the session cookie.

But then vary based on where the user ends up. If it's accounts, they end up back on the 503 error page, and the params are ignored.

if it's transition, then fine they can just end up back there.